README.md

```diff
@@ -14,7 +14,7 @@
 
 #### New Features
 
-- Can now specify that format run against a solution filter with `dotnet format solution.slnf`
+- Can now specify that format run against a solution filter with `dotnet format solution.sln`
```

Comment:
The extension in the example is intentional as solution filter files have the extension '.slnf'